### ISSUE TITLE
lightbox works on Chrome, but not on Firefox

### ISSUE THREAD

**Issue by sandboxmeg** [archetyped/simple-lightbox#1015]
## Description of Problem

The lightbox feature is working correctly in Chrome, but not in Firefox. Not sure why, please help?

## Details

* URL of Page exhibiting problem: https://teelin.com/feis/gallery/
* Browser(s) exhibiting problem (Firefox, Chrome, etc.): Firefox
* Version of SLB Installed: 2.9.3
* Other plugins installed: BackupBuddy, Font Awesome, GenerateBlocks, GenerateBlocks Pro, GP Premium, Lightweight Social Icons, SiteGround Optimizer, Smash Balloon Custom Facebook Feed, Smash Balloon Instagram Feed

**Comment by archetyped** [comment]
Hi, SLB is being prevented from working at the link provided on all browsers tested (including Firefox and Chrome).  You may need to clear your browser's cache to ensure it is not loading an outdated version of the site.

There appears to be another plugin causing a conflict that is preventing SLB from functioning.  Here are instructions for identifying the conflicting plugin](https://github.com/archetyped/simple-lightbox/wiki/Troubleshooting-Plugin-Conflicts).

Based on your installed plugins, I would recommend starting with temporarily disabling SiteGround Optimizer, as that plugin has been known to cause conflicts with the loading of other plugins.  If disabling that plugin resolves the issue, then excluding SLB's files from being processed by it ([documentation](https://www.siteground.com/tutorials/wordpress/sg-optimizer/frontend-optimization/)) should resolve the issue.

Please let me know what you find and I'll see if anything can be done from SLB's end.

**Comment by sandboxmeg** [comment]
Thanks very much for the help. Yes, disabling SiteGround Optimizer solved the problem. However, when I reactivated it, excluding SLB's files didn't seem to resolve the issue. Unless I'm missing another place to exclude?? Here is a screenshot of the two places that I found options to exclude SLB files:
<img width="799" alt="Screen Shot 2023-06-09 at 1 49 11 PM" src="https://github.com/archetyped/simple-lightbox/assets/136006859/0a9570ae-8fd7-44d6-b05e-3aa80d86f93b">
